content/webapp/components/Exhibition/Exhibition.tsx

```diff
@@ -303,7 +312,7 @@ const Exhibition: FunctionComponent<Props> = ({
                       >
                         {link.type === 'exhibition-guide' && (
                           <h3 className={font('intb', 4)}>
-                            Digital exhibition guide
+                            Exhibition display guide
```

Comment:
Was this discussed somewhere, I might've missed it and just want to check it's not a test-change!